doc/classes/ProjectSettings.xml

```diff
@@ -169,6 +169,7 @@
 			</return>
 			<description>
 				Saves the configuration to the [code]project.godot[/code] file.
+				[b]Note:[/b] This shouldn't be used in exported project, as modified [ProjectSettings] can't be loaded back in standalone app.
```

Comment:
```suggestion
				[b]Note:[/b] This shouldn't be used in exported projects, as modified [ProjectSettings] can't be loaded back in the running app.
```

Is it really the problem though, or is it simply that the `res://` is read-only so it's technically not possible to overwrite `project.godot` (which is actually `project.binary` once exported)?

Reply:
`ProjectSettings.save()` will create a `project.godot` next to your executable and it can't be loaded after #42980. We could add a method to load it manually if there's demand.

Reply:
Ah so this PR broke the feature, we didn't think about this use case being supported by `ProjectSettings.save()`.

Maybe clarify that this method is meant for use by editor plugins, and point to `save_custom` and the relevant settings override project settings as options for saving custom settings?

Reply:
Ok, changed the description. I also added possibility to save `override.cfg` with `save_custom()`, which makes it possible to change project settings in exported projects.